tests/system/python/packages/test_north_pi_webapi_nw_throttle.py

```diff
@@ -329,7 +329,7 @@ def test_omf_in_impaired_network(self, clean_setup_fledge_packages, reset_fledge
             raise Exception("None of packet delay or rate limit given, "
                             "cannot apply network impairment.")
         # Insert some readings before turning off compression.
-        time.sleep(2)
+        time.sleep(3)
```

Comment:
May be better safe to set 5s

Reply:
Anything change in Core or Plugin itself? Seems like it is required for system test only for ingestion of readings?
We have a fixture named as **wait_time** and having default `5s` value.